Assets/UnityShell/Editor/Scripts/UnityShellEditorWindow.cs

```diff
@@ -11,16 +11,24 @@ public class UnityShellEditorWindow : EditorWindow
 		static class Styles
 		{
 			public static readonly GUIStyle textAreaStyle;
-
+			// Default background Color(0.76f, 0.76f, 0.76f)
+			public static Color BgColorLightSkin = new Color(0.87f, 0.87f, 0.87f);

```

Comment:
You should make these variables readonly (`public static readonly Color BgColorLightSkin`) and maybe even private as well, and also add a blank line after `public static readonly GUIStyle textAreaStyle;` to keep these seperate.

Also, why do the comments for the BgColorLightSkin and BgColorDarkSkin have a different number?

Reply:
Hi, thanks a lot for reviewing the PR, I totally agree with all suggestions.

The comments for BgColorLightSkin and BgColorDarkSkin are different from the used values, because they are the default values unity uses for windows. It is the background color for a window.
I set the light skin color lighter and the dark skin color darker to get a visual difference from the window to give it a text editor feeling, instead of a non-interactable window. Since the default values are not very well documented I though including them might help.